data/json/recipes/recipe_food.json

```diff
@@ -59,6 +59,44 @@
   ]
 },{
   "type" : "recipe",
+  "result": "salt",
+  "id_suffix": "from_mineral",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_OTHER",
+  "skill_used": "cooking",
+  "difficulty": 4,
+  "skills_required": [ "cooking", 4 ],
+  "time": 90000,
+  "reversible": false,
+  "autolearn": false,
+  "book_learn": [["atomic_survival", 4], ["textbook_chemistry", 5], ["adv_chemistry", 4]],
+  "batch_time_factors":[80, 4],
+  "qualities":[
+    {"id":"BOIL","level":1,"amount":1}
+  ], "tools": [
+    [
+      [ "hotplate", 3 ],
+      [ "char_smoker", 1 ],
+      [ "toolset", 3 ],
+      [ "coffeemaker", 3],
+      [ "atomic_coffeepot", -1 ],
+      [ "fire", -1 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "material_rocksalt", 2 ]
+    ],
+    [
+      [ "water", 1 ],
+      [ "water_clean", 1 ]
+    ],
+    [
+      [ "iodine_solution", 1 ]
```

Comment:
This is going to get called out the first time a player has a pile of rock salt and no iodine; we don't otherwise differentiate types of salt, and this requirement makes production via this method pointless due to the added material requirement. Please lose the iodine requirement, as we don't actually track whether salt is iodized in DDA.